clang/test/dpct/cmake_migation/case_009/input.cmake

```diff
@@ -0,0 +1,10 @@
+CMake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.26)
+cmake_minimum_required(VERSION 3.10...3.26)
+set  (  VER 
+  3.15)
+set(MIN_VER 3.10)
```

Comment:
Please also consider add test case:  put set(MIN_VER 3.10) into separate file.  Please address it in a follow up PR.

Reply:
okay